code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -331,4 +332,13 @@
 		var/mob/living/carbon/human/H = synd_mind.current
 		synd_mind.name = H.dna.species.random_name(H.gender,0,lastname)
 		synd_mind.current.real_name = synd_mind.name
-	return
\ No newline at end of file
+	return
+
+/datum/game_mode/nuclear/setup_lowpop()
+	required_enemies = 2
+	recommended_enemies = 2
+	agents_possible = 2
+	max_players = 2

```

Comment:
if there are more than two players the mode won't run...
was this your intention? for testing?